common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -335,6 +323,118 @@
                 });
 
             });
+    </script>
+}
+
+@bootSystemJs() = {
+
+    <script>
+        @Html(Static.js.es6ModuleLoader)
+        @Html(Static.js.systemJs)
+        @Html(Static.js.systemJsAppConfig)
+
+        @if(!play.Play.isDev()) {
+            @Html(Static.js.systemJsBundleConfig)
+        }
+
+        System.paths['ophan/ng']  = '@{Configuration.javascript.config("ophanJsUrl")}.js';
+        System.paths['googletag'] = '@{Configuration.javascript.config("googletagJsUrl")}';
+
+        @Html(Static.js.systemJsNormalize)
+
+        window.require = System.amdRequire.bind(System);
```

Comment:
The fix for this is in System 0.16.8 and 0.16.9 https://github.com/systemjs/systemjs/releases/tag/0.16.9